kafka-statefulsets/resources/kafka.yaml

```diff
@@ -11,15 +11,22 @@ spec:
       labels:
         name: kafka
     spec:
+      securityContext:
+        fsGroup: 1001
+      initContainers:
+       - name: volume-mount-hack
+         image: busybox
+         command: ["sh", "-c", "chown -R 1001:1001 /tmp/kafka/"]
```

Comment:
Why are we putting the kafka data in `/tmp`, would somwhere under `/var` be more conventional?

Reply:
Both Zookeeper and Kafka data dirs are not in `/var/zookeeper` and `/var/kafka`.